mappings/net/minecraft/server/SaveLoader.mapping

```diff
@@ -6,4 +6,7 @@ CLASS net/minecraft/class_6904 net/minecraft/server/SaveLoader
 	METHOD comp_358 dynamicRegistryManager ()Lnet/minecraft/class_5455$class_6890;
 	METHOD comp_359 saveProperties ()Lnet/minecraft/class_5219;
 	METHOD method_40431 ofLoaded (Lnet/minecraft/class_7237$class_6906;Lnet/minecraft/class_7237$class_6907;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
This is more accurately
```suggestion
	METHOD method_40431 load (Lnet/minecraft/class_7237$class_6906;Lnet/minecraft/class_7237$class_6907;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
```

Reply:
Okay seems like `load` can be used both in static and non-static context (existing usage in resources etc). Renaming

Reply:
My rationale is that this works much like `SimpleResourceReload.create` where we return a future that has already been scheduled to the prepare and apply executors.